docs/changelog/105298.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 105298
+summary: Ignore duplicate `FieldAliasMappers`
+area: TSDB
+type: bug
```

Comment:
This is a bug in unreleased code, let's mark it as non-issue and then a changelog isn't needed.

Reply:
Done.